spark/src/test/scala/org/apache/spark/sql/delta/FindLastCompleteCheckpointSuite.scala

```diff
@@ -117,11 +139,15 @@ class FindLastCompleteCheckpointSuite
         commitFiles(logPath, 0L to 3000) ++
           singleCheckpointFiles(logPath, Seq(100, 200, 1000, 2000))
       )
-      assert(
-        log.findLastCompleteCheckpointBefore(Some(CheckpointInstance(version = 2000)))
-          .contains(CheckpointInstance(version = 1000)))
+      val eventData1 = getLastCompleteCheckpointUsageLog {
+        assert(
+          log.findLastCompleteCheckpointBefore(Some(CheckpointInstance(version = 2000)))
+            .contains(CheckpointInstance(version = 1000)))
+      }
       assert(logStore.listFromCount == 1)
       assert(logStore.elementsConsumedFromListFromIter == 1002 + 2) // commits + checkpoint
+      assert(eventData1("iterations") == "1")
+      assert(eventData1("numFilesScanned") == "1004")
```

Comment:
Does this work for better error messages?
```suggestion
      assert(eventData1("iterations") === "1")
      assert(eventData1("numFilesScanned") === "1004")
```

(bunch of others, including existing code... maybe worth a cleanup PR some day)

Reply:
For consistency in the test suite, I am keeping these as `==`. Will open a PR which converts all `==` to `===`.